File: ChatSupport/src/main/java/com/vantagecircle/chatapp/core/model/DataModel.java
package com.vantagecircle.chatapp.core.model;

import com.google.android.gms.internal.xf;
import com.google.firebase.database.DataSnapshot;
import com.google.firebase.database.DatabaseError;
import com.google.firebase.database.DatabaseReference;

/**
 * Created by bapidas on 01/08/17.
 */

public class DataModel {
    private DataSnapshot dataSnapshot;
    private DatabaseError databaseError;
    private String dataString;
    private String extraString;

    public DataSnapshot getDataSnapshot() {
        return dataSnapshot;
    }

    public void setDataSnapshot(DataSnapshot dataSnapshot) {
        this.dataSnapshot = dataSnapshot;
    }

    public DatabaseError getDatabaseError() {
        return databaseError;
    }

    public void setDatabaseError(DatabaseError databaseError) {
        this.databaseError = databaseError;
    }

    public String getDataString() {
        return dataString;
    }

    public void setDataString(String dataString) {
        this.dataString = dataString;
    }

    public String getExtraString() {
        return extraString;
    }

    public void setExtraString(String extraString) {
        this.extraString = extraString;
    }
}

File: app/src/main/java/com/bapidas/chattingapp/data/core/model/DataModel.kt
package com.bapidas.chattingapp.data.core.model

import com.google.firebase.database.DataSnapshot
import com.google.firebase.database.DatabaseError

/**
 * Created by bapidas on 01/08/17.
 */
data class DataModel(
        var dataSnapshot: DataSnapshot? = null,
        var databaseError: DatabaseError? = null,
        var dataString: String = "",
        var extraString: String = ""
)